File: MyBlog.API/Models/BlogPost.cs
using System;

namespace MyBlog.API.Models
{
    public class BlogPost
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public string Excerpt { get; set; }
        public string Contents { get; set; } 
        public DateTime DateCreated { get; set; }
        public DateTime DateModified { get; set; }
        public string Status { get; set; }

        
    }
}

File: MyBlog-SPA/src/app/_models/blogpost.ts
import { BlogPostCategories } from './BlogPostCategories';

export interface Blogpost {

    id: number;
    title: string;
    contents: string;
    dateCreated: Date;
    dateModified: Date;
    excerpt: string;
    status: string;
    categories: {
        id: number,
        name: string
    }
}
